libwild/src/aarch64.rs

```diff
@@ -895,6 +888,10 @@ impl crate::arch::Relaxation for () {
     fn debug_kind(&self) -> impl std::fmt::Debug {
         todo!()
     }
+
+    fn next_modifier(&self) -> RelocationModifier {
+        todo!()
```

Comment:
It should be fine to just return `RelocationModifier::Normal` here, since we don't yet implement any relaxations for aarch64